example/ionic-angular/android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,27 @@
+package com.getcapacitor.myapp;
```

Comment:
If you don't plan to have these instrumented tests on the example app, maybe we can remove them to have less files around